deploy-subgraph.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+set -e
+
+LOG_START='\n\e[1;36m' # new line + bold + color
+LOG_END='\n\e[0m' # new line + reset color
+DONE_START='\n\e[1;32m' # new line + bold + green
+DONE_END='\n\n\e[0m'    # new line + reset
+
+printf "${LOG_START}Fetching 'deploy-subgraph' npm packages...${LOG_END}"
+
+yarn
+
+printf "${LOG_START}Installing Graph CLI...${LOG_END}"
+yarn global add @graphprotocol/graph-cli
+
+# for CI purposes, it will get the latest deployed npm packages for the current CI jobs
+if [[ $KEEP_CORE_VERSION != "" ]] && [[ $KEEP_ECDSA_VERSION != "" ]] && [[ $KEEP_TBTC_VERSION != "" ]]
```

Comment:
What will happen if any of these variables is empty?

Reply:
It will take the Ropsten versions, but please see this [comment](https://github.com/keep-network/keep-subgraph/pull/1#discussion_r706135328)